advanced_functionality/scikit_bring_your_own/scikit_bring_your_own.ipynb

```diff
@@ -493,6 +497,53 @@
    "source": [
     "sess.delete_endpoint(predictor.endpoint)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Run Batch Transform Job\n",
+    "One can use trained model to get inference on data by using Amazon SageMaker Batch Transform. A Batch Transform Job takes your input data S3 location and publishes prediction result to specified S3 output folder. Similar to hosting, lets extract inferences for training data to demo Batch Transform mecahnism. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Initialize the transformer object\n",
+    "transform_output_folder = \"batch-transform-output\"\n",
+    "transformer = tree.transformer(instance_count=1,\n",
+    "                               instance_type='ml.m4.xlarge',\n",
+    "                               output_path=\"s3://{}/{}\".format(sess.default_bucket(), transform_output_folder))\n",
+    "\n",
+    "# Start a transform job:\n",
+    "transformer.transform(data_location, content_type='text/csv', split_type='Line')\n",
+    "\n",
+    "# Wait until transform job is completed\n",
```

Comment:
Instead of comments, create markdown between multiple cells. Please, be more detail on how to use batch transform and add links for additional reference as well.